app/Models/Game.php

```diff
@@ -367,27 +346,29 @@ public function getImageIngameUrlAttribute(): string
         return media_asset($this->ImageIngame);
     }
 
-    public function getIsSubsetGameAttribute(): bool
+    public function getParentGameIdAttribute(): ?int
     {
-        if (str_contains($this->title, "[Subset")) {
-            return true;
-        }
```

Comment:
I eliminated this check, but it could be used as a short-circuit for the `isSubsetGameAttribute` (still here, just lower in diff because the body got moved into another function). But as most games won't match the substring, you could argue it actually adds performance impact most of the time.